sources/gdal/large_image_source_gdal/__init__.py

```diff
@@ -561,7 +561,13 @@ def getBandInformation(self, statistics=True, dataset=None, **kwargs):
                     if band.GetMaskBand():
                         info['maskband'] = band.GetMaskBand().GetBand() or None
                     # Only keep values that aren't None or the empty string
-                    infoSet[i + 1] = {k: v for k, v in info.items() if v not in (None, '')}
+                    infoSet[i] = {
```

Comment:
Irritatingly, we have to maintain the 1-based indexing here (revert the `[i]` back to `[i + 1]`).

Reply:
Ok, I had wondered what the purpose was, but I see now. I put that back in [a225393](https://github.com/girder/large_image/pull/1414/commits/a225393f8068b4030f463b34d7823b51772ebbd3).